go.sum

```diff
@@ -48,8 +50,6 @@ github.com/armon/go-metrics v0.3.10 h1:FR+drcQStOe+32sYyJYyZ7FIdgoGGBnwLl+flodp8
 github.com/armon/go-metrics v0.3.10/go.mod h1:4O98XIr/9W0sxpJ8UaYkvjk10Iff7SnFrb4QAOwNTFc=
 github.com/armon/go-radix v0.0.0-20180808171621-7fddfc383310/go.mod h1:ufUuZ+zHj4x4TnLV4JWEpy2hxWSpsRywHrMgIH9cCH8=
 github.com/armon/go-radix v1.0.0/go.mod h1:ufUuZ+zHj4x4TnLV4JWEpy2hxWSpsRywHrMgIH9cCH8=
-github.com/benbjohnson/go-sqlite3 v0.0.0-20220723145740-eed275a583e0 h1:d37HZoNk/Z8Ds3vOS1N/rGpARaVic6shnKgxmqtVupE=
-github.com/benbjohnson/go-sqlite3 v0.0.0-20220723145740-eed275a583e0/go.mod h1:NyWgC/yNuGj7Q9rpYnZvas74GogHl5/Z4A/KQRfk6bU=
```

Comment:
these lines were removed after running `go mod tidy`, let me know if you want them back for any reason